middleman_protocol/middleman_protocol/helpers.py

```diff
@@ -0,0 +1,18 @@
+from asyncio import StreamReader
```

Comment:
Please move to `stream.py` or `stream_async.py`.

Reply:
done

Reply:
done